### PR TITLE
OTE_SDK. Increase code coverage label_schema.py, train_parameters.py

### PR DESCRIPTION
**Pull request by saltykox**
Test list:
•	ote_sdk/ote_sdk/tests/entities/test_label_schema.py (renamed from ote_sdk/ote_sdk/tests/entities/test_label_schema_entity.py)
•	ote_sdk/ote_sdk/tests/entities/test_train_parameters.py

Task: CVS-72107

### CODE REVIEW

**Comment by saltykox** [comment]
run ote_sdk tests

**Comment by saltykox** [comment]
@LeonidBeynenson please review

File: ote_sdk/ote_sdk/tests/entities/test_train_parameters.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    @pytest.mark.priority_medium
+    @pytest.mark.component
+    @pytest.mark.reqids(Requirements.REQ_1)
+    def test_train_parameters(self):
+        """
+        <b>Description:</b>
+        Check TrainParameters class object attributes
+
+        <b>Expected results:</b>
+        Test passes if attributes of TrainParameters class object are equal expected
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_train_parameters.py]:
```suggestion
        Test passes if attributes of TrainParameters class object are equal to expected
```

File: ote_sdk/ote_sdk/tests/entities/test_train_parameters.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+        <b>Steps</b>
+        1. Check attributes of TrainParameters object initiated with default parameters
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_train_parameters.py]:
```suggestion
        1. Check attributes of TrainParameters object initialized with default parameters
```

File: ote_sdk/ote_sdk/tests/entities/test_train_parameters.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        2. Check attributes of TrainParameters object initiated with "resume", "update_progress" and "save_model"
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_train_parameters.py]:
```suggestion
        2. Check attributes of TrainParameters object initialized with "resume", "update_progress" and "save_model"
```

File: ote_sdk/ote_sdk/tests/entities/test_train_parameters.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        parameters
+        """
+        # Checking attributes of TrainParameters object initiated with default parameters
+        default_values_train_parameters = TrainParameters()
+        assert not default_values_train_parameters.resume
+        # Expected that update_progress equal to function for default parameters TrainParameters object
+        assert (
+            default_values_train_parameters.update_progress == default_progress_callback
+        )
+        # Expected that save_model equal to function for default parameters TrainParameters object
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_train_parameters.py]:
```suggestion
        # Expected that save_model is equal to function for default parameters TrainParameters object
```

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py [hunk 5]
```diff
@@ -0,0 +1,2169 @@
...
+        label_tree.add_child(benign, bottle)
+
+        assert label_tree.get_parent(gun) == threat
+        assert label_tree.get_parent(rifle) == threat
+        assert label_tree.get_parent(bottle) == benign
+        assert label_tree.get_parent(threat) is None
+
+        threat_children = label_tree.get_children(threat)
+        assert rifle in threat_children
+        assert rifle in threat_children
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label_schema.py]: redundant

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py [hunk 6]
```diff
@@ -0,0 +1,2169 @@
...
+        Tests a mix of hierarchy and exclusivity
+
+        <b>Input data:</b>
+        Exclusive group: flowering,vegetative,no_plant
+        Exclusive group: flower_partial_visible,flower_fully_visible (children of flowering)
+        Exclusive group: few_leaves (child of vegetative)
+        Exclusive group: bee
+
+        <b>Expected results:</b>
+        Test passes if the LabelSchema and hierarchy are correctly represented
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label_schema.py]:
```suggestion
        Test passes if the LabelSchemaEntity and hierarchy are correctly represented
```

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py [hunk 7]
```diff
@@ -0,0 +1,2169 @@
...
+
+        <b>Steps</b>
+        1. Create LabelSchema with hierarchy
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label_schema.py]:
```suggestion
        1. Create LabelSchemaEntity with hierarchy
```

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py [hunk 8]
```diff
@@ -0,0 +1,2169 @@
...
+        <b>Input data:</b>
+        Exclusive Group "Vegetation"
+        LabelSchemaExample instance
+
+        <b>Expected results:</b>
+        Test passes if a single label and multiple labels can be added to the exclusive group
+        and the correct labels can be retrieved
+
+        <b>Steps</b>
+        1. Create LabelSchema with "Vegetation" exclusive group
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label_schema.py]:
```suggestion
        1. Create LabelSchemaEntity with "Vegetation" exclusive group
```

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py [hunk 9]
```diff
@@ -0,0 +1,2169 @@
...
+        assert label_schema_example.no_plant in exclusive_to_flowering
+        assert label_schema_example.vegetative in exclusive_to_flowering
+
+    @pytest.mark.priority_medium
+    @pytest.mark.component
+    @pytest.mark.reqids(Requirements.REQ_1)
+    def test_labelschema_equality(self, label_schema_example):
+        """
+        <b>Description:</b>
+        Check that LabelSchema equality works correctly
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label_schema.py]:
```suggestion
        Check that LabelSchemaEntity equality works correctly
```

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py [hunk 10]
```diff
@@ -0,0 +1,2169 @@
...
+
+        <b>Input data:</b>
+        LabelSchema instances
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label_schema.py]:
```suggestion
        LabelSchemaEntity  instances
```

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py [hunk 11]
```diff
@@ -0,0 +1,2169 @@
...
+
+        <b>Expected results:</b>
+        == and != operations work correctly for various inputs
+
+        <b>Steps</b>
+        1. Test LabelSchema equality
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label_schema.py]:
```suggestion
        1. Test LabelSchemaEntity equality
```

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py [hunk 12]
```diff
@@ -0,0 +1,2169 @@
...
+    @pytest.mark.priority_medium
+    @pytest.mark.component
+    @pytest.mark.reqids(Requirements.REQ_1)
+    def test_label_tree(self, label_schema_example):
+        """
+        <b>Description:</b>
+        Check that childs and parents of Labels can be found correctly
+
+        <b>Input data:</b>
+        A simple Label Tree
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label_schema.py]:
```suggestion
        A simple LabelTree
```

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py [hunk 13]
```diff
@@ -0,0 +1,2169 @@
...
+    @pytest.mark.priority_medium
+    @pytest.mark.component
+    @pytest.mark.reqids(Requirements.REQ_1)
+    def test_label_group_type(self):
+        """
+        <b>Description:</b>
+        Check LabelGroupType Enum class elements
+
+        <b>Expected results:</b>
+        Test passes if LabelGroupType Enum class length is equal expected value and its elements have expected
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label_schema.py]:
```suggestion
        Test passes if LabelGroupType Enum class length is equal to expected value and its elements have expected
```

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py [hunk 14]
```diff
@@ -0,0 +1,2169 @@
...
+        """
+        <b>Description:</b>
+        Check remove_label method of LabelGroup class object
+
+        <b>Input data:</b>
+        LabelGroup objects with specified name, labels, group type and id_parameters parameters
+
+        <b>Expected results:</b>
+        Test passes if after using remove_label method values of "labels" property, "minimum_label_id" and
+        "is_single_label" methods are equal expected
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label_schema.py]:
```suggestion
        "is_single_label" methods are equal to expected
```

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py [hunk 15]
```diff
@@ -0,0 +1,2169 @@
...
+    def test_label_graph_initialization(self):
+        """
+        <b>Description:</b>
+        Check LabelGraph class object initialization
+
+        <b>Input data:</b>
+        LabelGraph objects with specified "directed" parameter and added edges
+
+        <b>Expected results:</b>
+        Test passes if LabelGraph object "directed" attribute and "num_labels" and "type" properties are equal expected
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label_schema.py]:
```suggestion
        Test passes if LabelGraph object "directed" attribute and "num_labels" and "type" properties are equal to expected
```

**Review by Ilya-Krylov — CHANGES_REQUESTED** [review]

File: ote_sdk/ote_sdk/tests/entities/test_label_schema.py (hunk 5 above)
Comment (saltykox) [reply ote_sdk/ote_sdk/tests/entities/test_label_schema.py]: Thanks, removed redundant assertion from test_label_tree

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Pull request merged**